Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md

```diff
@@ -0,0 +1,78 @@
+**Critical RCE Vulnerability: log4j - CVE-2021-44228**
+
+On Dec. 9, 2021, a remote code execution (RCE) vulnerability in Apache log4j 2 was identified being exploited in the wild. Public proof of concept (PoC) code was released and subsequent investigation revealed that exploitation was incredibly easy to perform. 
```

Comment:
...was identified **_as_** being exploited in the wild.

Note: that this sentence is used in other areas of this pack

Reply:
For the grammar, I'm keeping it the same as in Unit42 blog.